pkg/catalog/catalog.go

```diff
@@ -122,6 +126,8 @@ const (
 	ListTagsLimitMax         = 1000
 	DiffLimitMax             = 1000
 	ListEntriesLimitMax      = 10000
+	sharedWorkers            = 15
+	workersToBufferRatio     = 3
```

Comment:
I don't really understand why we need to do the math here and not just define the actual buffer size we want to use